src/Misc/layoutbin/en-US/strings.json

```diff
@@ -138,6 +138,8 @@
   "NotLinux": "This Agent version is built for Linux. Please install a correct build for your OS.",
   "NotOSX": "This Agent version is built for OSX. Please install a correct build for your OS.",
   "NotWindows": "This Agent version is built for Windows. Please install a correct build for your OS.",
+  "NTLMPassword": "Password",
+  "NTLMUsername": "Domain\\Username",
```

Comment:
Actually just printing "User name" might be more accurate (since TFS doesn't require a domain).

Can you update the usage info as well? In the usage information we can give the example domain\username or UPN. There's verbiage with examples already on MSDN for the windows agent you can pull from as well.